components/project-loader/project-loader.jsx

```diff
@@ -54,6 +55,11 @@ export default React.createClass({
       return Object.assign(combinedParams, { page: 1 });
     }
 
+    if (combinedParams.moderationState) {
+      combinedParams.moderationstate = combinedParams.moderationState.label;
```

Comment:
This is a really bad idea. At the very least, rename this from `combinedParams.moderationstate` to `combinedParams.moderationStateLabel` so that it doesn't look like you're saying `x.a = x.a.b`. To anyone with even the mildest form of dyslexia, including someone who's not had their morning coffee yet, that case difference isn't there.